### PR TITLE
Improve the Prepare > Data Frame > Delete Columns/Rows dialogue 

### PR DESCRIPTION
**Pull request by Wycklife**
Fixes  part one of #5618

@africanmathsinitiative/developers This is ready for review

### CODE REVIEW

**Comment by lloyddewit** [comment]
I understand part one of #5618 requests that 'the controls with the row numbers become wider, so they can show row numbers up to 10 million at least'.
I see 3 row number controls ('number of rows', 'delete from' and 'to'). I understand that all 3 of these controls should be widened to show at least 8 digits.
@rdstern did I understand correct?
@Wycklife In the 'files changed' section I see that the 'number of rows' control has been widened but not the other 2 controls. Do the other 2 controls also need widening?

**Comment by Wycklife** [comment]
@lloyddewit  Thank you, I misunderstood this. I will work on the remaining two controls.

**Comment by rdstern** [comment]
@Wycklife this is an "interesting dialogue!
It is much better now.  But I took the (usual) survey data, then deleted rows 34 to 36.
Then - when I returned to the dialogue it opened fine - it used to give an error first.  Then the From still said 34 and the To (correctly) said 33.  There were now 33 rows - so pressing ok with the 34 - -which I did (of course) gave an error.
Also when I delete the From or To in the controls the OK is still enabled.  

So I tried to see what happens with deleting variables.  (And found another error, but it is close there.)
I deleted the last variable.  Then returned and it gave an error.  Then the receiver was empty - which is fine.  So - we need to trap that error.

I deleted another variable and it was fine - when I returned the control was empty.  I am happy with that.

So, back to the rows.  Can the up-down start empty?  If so, then I think it should.  And ok needs both to have a number in to be enabled.
So, if it ever becomes empty, then you disable ok.
Also =, it doesn't remember anything.  When you return to the dialogue the From and To are empty again.  Is that easy to do?

**Comment by Wycklife** [comment]
@rdstern This is done and now working well. Kindly confirm.

File: instat/ucrNud.vb [hunk 1]
```diff
@@ -145,13 +145,17 @@ Public Class ucrNud
         Return bTemp
     End Function
 
-    Public Function GetText() As String
-        Return CStr(Value)
+    Public Function GetText() As String
+        If nudUpDown.Text = "" Then
+            Return ""
+        Else
+            Return CStr(Value)
```
Comment (Patowhiz) [inline instat/ucrNud.vb:152]: This is trivial, but for future reference I'm mentioning it here. This is unnecessary cast. The `Value` which is a `Decimal` has a concrete implementation of a `ToString()` function which returns the `Decimal` as a `String`. This function is called implicitly by the compiler.

**Pull request merged**